src/main/java/org/jboss/ejb/protocol/remote/RemoteServer.java

```diff
@@ -416,7 +416,7 @@ private void writeFailedResponse(final int invId, final Exception e) {
             this.transaction = transaction;
         }
 
-        int getInvId() {
+        public int getInvId() {
```

Comment:
This is used in Association class implementation to trace incomming invocations on the server side.